mathics/builtin/numbers/hyperbolic.py

```diff
@@ -325,11 +325,11 @@ class Gudermannian(Builtin):
         "Gudermannian[0]": "0",
         "Gudermannian[2*Pi*I]": "0",
         # FIXME: handling DirectedInfinity[-I] leads to problems
-        # "Gudermannian[6/4 Pi I]": "DirectedInfinity[-I]",
+        "Gudermannian[6/4 Pi I]": "DirectedInfinity[-I]",
         # Handled already
-        # "Gudermannian[Infinity]": "Pi/2",
+        "Gudermannian[DirectedInfinity[1]]": "Pi/2",
```

Comment:
Notice that the rule requires to use `DirectedInfinity[1]` instead of `Infinity`. This is due to `Infinity` is evaluated to `DirectedInfinity[1]` before looking at the `Gudermannian`  set of rules.

Reply:
Ok - Thanks for the explanation.  I'd like to understand this more though. What is it that turns `Infinity` into `DirectedInfinity[1]` ? Is this SymPy doing this?

Reply:
We should if we want to get the same result if we apply the same rules as written for WMA.

Reply:
Ok. If this is simple for now to do, then let's do. Or we can write `DirectedInfinity[1]` in the rules and come back to this some other time. Your choice - assuming this is a simple fix.